SQL.js

```diff
@@ -35,6 +35,20 @@ class SqlStatement {
     return new SqlStatement(result.strings, result.values)
   }
 
+  /**
+   * A function that accepts an array of objects and a mapper function
+   * It returns a clean SQL format using the object properties defined in the mapper function
+   */
+  map (array, mapFunc) {
```

Comment:
I wonder if a default mapFunc equals to `i => i` could be even more easy to use.

Here a sample usage without the default:

```
// as is now
const query2 = SQL`SELECT * FROM table WHERE id IN (${SQL.map(ids, i=>i)})`

// with default:
const query2 = SQL`SELECT * FROM table WHERE id IN (${SQL.map(ids)})`
```

Reply:
Done, I have added a default mapper function